apps/chat-e2e/src/tests/promptMaximumNameLength.test.ts

```diff
@@ -0,0 +1,177 @@
+      0,
+      ExpectedConstants.maxEntityNameLength,
+    );
+    const nameUnder160Symbols =
+      'This prompt is renamed to very long-long-long name to see how the system cuts the name';
+
+    await dialTest.step(
+      'Create a prompt and enter text longer than 160 symbols',
+      async () => {
+        await dialHomePage.openHomePage();
+        await dialHomePage.waitForPageLoaded({
+          isNewConversationVisible: true,
+        });
+        await promptBar.createNewPrompt();
+        await promptModalDialog.setField(promptModalDialog.name, longName);
+        await promptModalDialog.setField(
+          promptModalDialog.prompt,
+          ExpectedConstants.newPromptTitle(1),
+        );
+      },
+    );
+
+    await dialTest.step('Save the prompt', async () => {
+      await promptModalDialog.saveButton.click();
+    });
+
+    await dialTest.step(
+      'Verify the prompt name is cut to 160 symbols and no error toast is shown',
+      async () => {
+        await promptAssertion.assertEntityState(
+          { name: expectedName },
+          'visible',
+        );
+        await errorToastAssertion.assertToastIsHidden();
+      },
+    );
+
+    await dialTest.step('Rename the prompt to a long name', async () => {
+      await prompts.openEntityDropdownMenu(expectedName);
+      await promptDropdownMenu.selectMenuOption(MenuOptions.edit);
+      await promptModalDialog.setField(
+        promptModalDialog.name,
+        nameUnder160Symbols,
+      );
+      // Wait for the API request to update the prompt name
+      await promptModalDialog.updatePromptDetailsWithButton(
+        nameUnder160Symbols,
+        prompt.description || '',
+        prompt.content || '',
+      );
+      prompt.name = nameUnder160Symbols;
+    });
+
+    await dialTest.step('Check the prompt name in the panel', async () => {
+      const promptNameElement = prompts.getPromptName(prompt.name);
+      const promptNameOverflow =
+        await promptNameElement.getComputedStyleProperty(Styles.text_overflow);
+      expect
+        .soft(promptNameOverflow[0], ExpectedMessages.entityNameIsTruncated)
+        .toBe(Overflow.ellipsis);
+    });
+
+    await dialTest.step(
+      'Hover over the prompt name and check the name in the panel',
+      async () => {
+        await prompts.getPromptName(prompt.name).hoverOver();
+        await promptAssertion.assertEntityDotsMenuState(
+          { name: prompt.name },
+          'visible',
+        );
+      },
+    );
+
+    await dialTest.step(
+      'Create two folders: Folder_parent -> Folder_child',
+      async () => {
+        for (let i = 1; i <= 2; i++) {
+          await promptBar.createNewFolder();
+          await expect
+            .soft(
+              folderPrompts.getFolderByName(
+                ExpectedConstants.newPromptFolderWithIndexTitle(i),
+              ),
+              ExpectedMessages.folderIsVisible,
+            )
+            .toBeVisible();
```

Comment:
can be replaced with: `await promptBarFolderAssertion.assertFolderState(ExpectedConstants.newPromptFolderWithIndexTitle(i))`

Reply:
done